### DISCUSSION TITLE
Proposal: Process for Becoming a Darwinia Community DAO Member

### DISCUSSION THREAD

**Discussion by furoxr** [ringdao-community-guild/discussion#1] [General]
> Proposal in [Snapshot](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xadf0533b8e283bbfacb61e69c83f6ca3b624d4de0c78a0a29883ebe6adc85bcb)

## Introduction

Every individual possessing a Profile NFT from the Darwinia Community DAO (DCDAO) is regarded as a member and has the ability to create proposals and cast votes on them. The objective of this proposal is to establish a transparent and straightforward process for contributors seeking to become members of the DCDAO.

## What is the Profile NFT

The Profile NFT is a non-fungible token that represents a unique identity on the ethereum. Its distribution involves three main steps: apply, audit, and release. This part explains the process and how contributors can apply for a Profile NFT.

### Step 1: Apply

To apply for a Profile NFT, a contributor must create a new application issue in the [Profile GitHub repository](https://github.com/dcdao/profiles). The issue should include the following information:

* Contributor's public address on the Ethereum

* Name and description of the profile

* Selected image

* Description of the contribution

### Step 2: Audit 

After a contributor applies for a Profile NFT, the application undergoes an audit process. The audit aims to verify that the application meets the following requirements:

* The contributor has a valid public address on the Ethereum

* The profile name and description are appropriate and not misleading

* Applicant’s contribution should be true and valid

### Step 3: Release

Once an application passes the audit process, the Profile NFT is released to the contributor's public address on the Darwinia Network. The Profile NFT contains the following information:

* Nickname

* Description

* Role

* Image

The profiles data is stored on two platforms: GitHub repository and IPFS. The GitHub repository hosts the application issues and the audit records. The IPFS stores the Profile NFT metadata, including the profile name, description, and image URL.

## Detail of the workflow

After a release branch is created and a draft PR is prepared in the Profile repository, applicants can create an application issue and fill out the necessary information. Reviewers will add an 'approved' label if they approve the application, and this label event will trigger a CI action to update the applicant's information in the release branch. Then the PR will be audited, and if all applications are approved, images and metadata will be uploaded to IPFS through the CI of GitHub, and the PR will be merged into the main branch. After multiple signatures of mint transactions, NFTs will be distributed to applicants.

## Detail of the audit

To clarify the process of audit:

* At the start of DCDAO, it is acceptable for the six founders to act as reviewers. If at least three of them approve an application, the corresponding pull request can be merged, and the applicants can be considered as official members.

* After more members have joined, any new applications must be audited through a proposal. Only after receiving enough votes of approval will the applicants be admitted and receive their Profile NFTs.

## Requirements

Contributors to Darwinia Network are all qualified to apply the Profile NFT and get the membership,  eg:

* Developers, managers and designers who has contributed to the development and maintenance of the network's underlying blockchain protocol, as well as developers who have built decentralized applications or tools on top of the network.

* Ambassadors who supports and promotes the adoption of Darwinia Network through community engagement, education, and collaboration with project teams

* Moderators who are responsible for managing and supervising community operations

* Researchers who are exploring new applications and development trends of blockchain technology related to Darwinia Network

* And so on…

**Comment by hackfisher** [comment]
LGTM

**Comment by 1022ranji** [comment]
Address on `Ethereum` or Address on `Darwinia Network`? The Other look good to me.
<img width="815" alt="image" src="https://user-images.githubusercontent.com/80877629/233968436-fabf7b89-a94f-4946-9329-1ac7bb6e24cd.png">

**Comment by hackfisher** [comment]
https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xadf0533b8e283bbfacb61e69c83f6ca3b624d4de0c78a0a29883ebe6adc85bcb
